lib/oauth2-client.js

```diff
@@ -103,7 +110,7 @@ export async function getNewAccessToken({
     }
   }
   throw new BedrockError(
-    `Service Unavailable`, 'InvalidStateError', {
+    `Service Unavailable for ${audience}.`, 'InvalidStateError', {
```

Comment:
I think this should be more like `Service Unavailable: Could not renew token for ${audience}`